src/client/modules/pageNotFound/containers/PageNotFound.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import Helmet from 'react-helmet';
 import { Button } from 'reactstrap';
-import PageLayout from '../../../app/page_layout';
+import PageLayout from '../../../app/PageLayout';
 
 const pageNotFound = () => (
```

Comment:
should we change this to `PageNotFound` to be consistent with imports of this component.

Reply:
That is the Page wrapper component, not `PageNotFound` module

Reply:
`PageNotFound.jsx` is not a `PageNotFound` module?

Reply:
I am commenting line 7.

Reply:
ah I see now. Yes, I will change that. I was paying attention to the line 5

Reply:
Sorry for the confusion.